jetty-http2/http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/Huffman.java

```diff
@@ -390,9 +394,20 @@ public static String decode(ByteBuffer buffer,int length)
         {
             int c = (current << (8 - bits)) & 0xFF;
             node = tree[node*256+c];
-            if (rowbits[node]==0 || rowbits[node] > bits) 
+
+            if (rowbits[node]==0 || rowbits[node] > bits)
+            {
+                int requiredPadding = 0;
+                for(int i=0; i<bits; i++)
+                    requiredPadding = (requiredPadding << 1) | 1;
+
+                if((c>>(8-bits)) != requiredPadding)
+                    throw new HpackException.StreamException("Incorrect padding");
```

Comment:
Is this both too much padding and incorrect padding?

Reply:
this is if the padding is not all 1's, the "too much padding" has not been implemented yet